src/test/java/org/opensearch/knn/jni/JNIServiceTests.java

```diff
@@ -749,6 +749,31 @@ public void testLoadIndex_nmslib_valid() throws IOException {
         assertNotEquals(0, pointer);
     }
 
+    public void testLoadIndex_nmslib_valid_with_stream() throws IOException {
```

Comment:
Do we have iTs as well? Will those be in another PR?

Reply:
We do have IT already. Since after this change, NMSLIB related IT will always read through Lucene's IndexInput.
And from local, passed them all.